gensim/interfaces.py

```diff
@@ -267,7 +263,7 @@ def __iter__(self):
                 # (unlike numpy). so, clip the end of the chunk explicitly to make
                 # scipy.sparse happy
                 chunk_end = min(self.index.shape[0], chunk_start + self.chunksize)
-                chunk = self.index[chunk_start : chunk_end]
+                chunk = self.index[chunk_start: chunk_end]
```

Comment:
PEP8: https://www.python.org/dev/peps/pep-0008/#other-recommendations (here and elsewhere).